Makefile

```diff
@@ -73,13 +72,22 @@ MKBIN_D = $(CXX) $(CXXFLAGS) $(FLAGS_D) $^ $(LDFLAGS) -o $@
 #------------------ Are you sure you know what you're doing? ------------------#
 #==============================================================================#
 
+# Checks
+
+ifeq ($(BLDDIR), .)
+  $(error "You MONSTER, never set BLDDIR to '.'. This clutters the directories and complicates cleaning.")
+endif
```

Comment:
The "and complicates cleaning" part is not really a good argument anymore, since our Makefile would handle it perfectly fine now, thanks to the hardcoded subdirectories ```obj``` and ```dir```. I would remove that part.

In fact, I just realized that the only reason to keep this test is to prevent the Makefile from removing empty directories in user-created ```obj``` and ```dir``` directories at the project root, an _extremely_ niche safety measure. We could remove the test entirely for simplicity, though it is funny.

Reply:
Even though I like the idea of insulting the user, I agree setting `BLDDIR` to `.` doesn't create a big enough problem to warrant an error message. I'm fine with removing this check.

Reply:
I changed the warning, but left in the check because it's just too funny.